backend/alert/serializers.py

```diff
@@ -5,6 +5,12 @@
 from alert.models import Registration
 from courses.models import Section, StatusUpdate, string_dict_to_html
 
+from django.db.models import Count, Max, Min
+
+from courses.util import get_current_semester
+
+from django.core.cache import cache
```

Comment:
Generally, all the `django` imports should go above all our first-party imports

Reply:
Yeah this isn't linted yet.